include/motor-control/core/motor_messages.hpp

```diff
@@ -2,8 +2,19 @@
 
 #include <cstdint>
 
+typedef int32_t sq0_31;  // 0: signed bit,  1-31: fractional bits
+typedef int64_t
+    sq32_31;  // 0: signed bit, 1-32: integer bits, 33-64: fractional bits
+
 namespace motor_messages {
+
+struct CanMove {
+    int32_t target_position;  // in mm
```

Comment:
we might want to reframe this; integer mm are almost certainly not the right unit here. either fixed- or floating-point mm, or integer microns are probably better.

Reply:
and let's make it unsigned